roles/common/defaults/main.yaml

```diff
@@ -1,14 +1,21 @@
 ---
 common_debug_massage: "hElLo tHeRe"
 
+# NTP Role
+ntp_package: ntp
```

Comment:
I am more than sure that these parameters will never change. So it's better to hardcode them
```yaml
ntp_package: ntp
ntp_service: ntpd
ntp_configfile: /etc/ntp.conf
ntp_configfile_user: root
ntp_configfile_group: root
```

Reply:
fixtd